moodlecli/utils.py

```diff
@@ -42,6 +42,22 @@ def generate_password(length=12):
     return ''.join(password)
 
 
+def generate_friendly_password():
+    adjectives = ["cold", "silly", "suave", "pensive", "lying", "pias",
```

Comment:
I'm just curious -- what does `pias` mean? (just stood out as I'm not familiar with it :sweat_smile: )

Reply:
That's funny I totally meant 'pious' as in devoutly religious, but had no lack of confidence in my spelling. I'll change it unless we think its fun to keep everyone on their toes! 😂 It COULD be a word.

Reply:
Ah -- let's use `pious` then, just in case `pias` can somehow be interpreted in an undesirable way by creative students :joy: